docs/content/docs/patterns/observability/metrics.md

```diff
@@ -138,6 +138,39 @@ osm metrics disable --namespace test
 osm metrics disable --namespace "test1, test2"
 ```
 
+## Custom Metrics
+To implement the [SMI metrics spec][7], OSM adds a custom WebAssembly extension to each Envoy proxy which generates the following statistics for HTTP traffic:
```

Comment:
We should support these metrics for tcp as well given that OSM now allows TCP traffic. It doesn't need to be solved in this PR but something to make note of

Reply:
I agree. For this initial implementation I'll plan to only include compatibility with HTTP. I've opened #2506 to track making it applicable to all TCP traffic.